float64 left_js_x
float64 left_js_y
float64 right_js_x
float64 right_js_y
float64 right_trigger
float64 left_trigger
